src/js/view/controls/components/timeslider.js

```diff
@@ -101,11 +101,13 @@ class TimeSlider extends Slider {
                     this.updateAriaText();
                 }
             })
-            .change('playlistItem', this.onPlaylistItem, this)
             .change('position', this.onPosition, this)
             .change('buffer', this.onBuffer, this)
             .change('streamType', this.onStreamType, this);
 
+        // Clear cues on player model's playlistItem change event
+        this._model.player.change('playlistItem', this.onPlaylistItem, this);
```

Comment:
Looked into using `on`, but it is not triggered on the initial setup, resulting `this.onPlaylistItem` to not trigger on the 1st playlist item